src/schema/prov.yaml

```diff
@@ -20,7 +20,9 @@ default_curi_maps:
 
 imports:
   - basic_slots
+  - core
   - linkml:types
```

Comment:
`linkml:types` necessary? It looks like core imports it too already

Reply:
good point